app/models/application.rb

```diff
@@ -2,4 +2,7 @@ class Application < ApplicationRecord
   include TenancyConcern
   belongs_to :source
   belongs_to :application_type
+
+  attribute :availability_status, :string
```

Comment:
Any reason we need to set the `cast_type` on these strings?

Reply:
I copied what was already in [Source](https://github.com/ManageIQ/sources-api/blob/master/app/models/source.rb#L9)
We can change it but would like to keep it consistent across the board